### PR TITLE
[pt-BR] AppStore Description: Translate some strings

### PR DESCRIPTION
**Pull request by FabioBeneditto**
Translate some strings. Needs revision.

### CODE REVIEW

File: pt-BR/apple_description_release.lang [hunk 1]
```diff
@@ -46,11 +46,11 @@ Spend your time reading your favorites sites instead of looking for them.
 
 # Slogan in uppercase, displayed as a category title
 ;SYNC FIREFOX ACROSS YOUR DEVICES
-SYNC FIREFOX ACROSS YOUR DEVICES
+TODOS SEUS DEVICES SINCRONIZADOS COM FIREFOX SYNC
```
Comment (reuben) [inline pt-BR/apple_description_release.lang]: "Dispositivos" em vez de "devices".

File: pt-BR/apple_description_release.lang [hunk 2]
```diff
@@ -46,11 +46,11 @@ Spend your time reading your favorites sites instead of looking for them.
...
 
 
 ;With a Firefox Account, access your history and open tabs everywhere you use Firefox.
-With a Firefox Account, access your history and open tabs everywhere you use Firefox.
+Com sua Conta Firefox, acesse seu histórico e abas abertas em qualquer lugar que usar Firefox.
```
Comment (reuben) [inline pt-BR/apple_description_release.lang:53]: "Firefox Account" em vez de "Conta Firefox".

**Comment by reuben** [comment]
r=me with those fixed. Thanks for the translations @FabioBeneditto !

File: pt-BR/apple_description_release.lang (hunk 2 above)
Comment (flodolo) [inline pt-BR/apple_description_release.lang:53]: Firefox Account seems to <a href="https://transvision.mozfr.org/?recherche=Firefox+Account&repo=aurora&sourcelocale=en-US&locale=pt-BR&search_type=strings_entities">be translated</a> in Portuguese.

Comment (FabioBeneditto) [inline pt-BR/apple_description_release.lang:53]: No Firefox desktop, a expressão está traduzida para "Conta Firefox". Embora seja um serviço, pensei ser a melhor opção. Mas, posso estar enganado.

Comment (reuben) [inline pt-BR/apple_description_release.lang:53]: You guys are right, of course. I got this confused with "Firefox Sync". You can merge with the string like this.

**Comment by flodolo** [comment]
I had to manually fix some conflicts, since in the meantime some strings were translated. Please double check
https://github.com/mozilla-l10n/appstores/commit/856e65e517b5f4cabb4d0a936643f7845bd85a83

@FabioBeneditto Can you get in touch with the team? I'm sure they'll be able to get help, and you can figure out the best way to work with them
https://wiki.mozilla.org/L10n:Teams:pt-BR

**Pull request merged**